oleesir/diaryReload
# #160920719 user should be able to see reminder page

#### What does this PR do?
This pull request adds a reminder template to the app

#### Description of Task to be completed?
* add reminder template

#### How should this be manually tested?
* Navigate to the UI folder and open reminder.html in your browser
* You will see a reminder template

#### What are the relevant pivotal tracker stories?
[#160920719](https://www.pivotaltracker.com/story/show/160920719)

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-10-03 at 4 29 39 pm" src="https://user-images.githubusercontent.com/17679511/46421269-afb10280-c729-11e8-8039-8ceffecfbf81.png">
